core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/script/ScriptLabelConfiguration.java

```diff
@@ -100,40 +104,43 @@ public void insert(ScriptLabel scriptLabel) {
         if (exists(scriptLabel)) {
             throw new MetadataAlreadyExistsException(scriptLabel);
         }
-        getMetadataRepository().executeUpdate("INSERT INTO " + getMetadataRepository().getTableNameByLabel("ScriptLabels") +
-                " (ID, SCRIPT_ID, SCRIPT_VRS_NB, NAME, VALUE) VALUES (" +
+        getMetadataRepository().executeUpdate( "INSERT INTO " + getMetadataRepository().getTableNameByLabel("ScriptLabels") +
+                " (ID, SCRIPT_ID, SCRIPT_VRS_NB, NAME, VALUE, DELETED_AT) VALUES (" +
                 SQLTools.getStringForSQL(scriptLabel.getMetadataKey().getId()) + "," +
                 SQLTools.getStringForSQL(scriptLabel.getScriptKey().getScriptId()) + "," +
                 SQLTools.getStringForSQL(scriptLabel.getScriptKey().getScriptVersion()) + "," +
                 SQLTools.getStringForSQL(scriptLabel.getName()) + "," +
                 SQLTools.getStringForSQLClob(scriptLabel.getValue(),
                         getMetadataRepository().getRepositoryCoordinator().getDatabases().values().stream()
                                 .findFirst()
-                                .orElseThrow(RuntimeException::new)) + ");");
-
+                                .orElseThrow(RuntimeException::new)) + ", 'NA' );");
     }
 
     public boolean exists(ScriptLabelKey scriptLabelKey) {
         String queryScriptParameter = "select ID, SCRIPT_ID, SCRIPT_VRS_NB, NAME, VALUE from " + getMetadataRepository().getTableNameByLabel("ScriptLabels")
-                + " where ID = " + SQLTools.getStringForSQL(scriptLabelKey.getId()) + ";";
+                + " where ID = " + SQLTools.getStringForSQL(scriptLabelKey.getId()) + " AND " +
+                " DELETED_AT = 'NA' ;";
```

Comment:
DELETED_AT should not be present in this query

Reply:
If we won't use DELETED_AT, then it will through **MetadataAlreadyExistsException** while adding new record after soft deletion